---
layout: page
title: Research
permalink: /research/
---

<style>
div {
  text-align: justify;
  text-justify: inter-word;
}
</style>

<figure style="float:right;margin-left:20px;margin-bottom:30px;height:284px;width:213px;text-align:center">
	<img src="../images/Cichaz stocktanks.jpg" alt="Stocktanks at the CICHAZ field station">
	<figcaption><i>Xiphophorus stocktanks at CICHAZ field station</i></figcaption>
</figure>
<div>
Getting involved with research in evolutionary biology has been a long time dream of mine, and I feel extremely fortunate that I have been gifted so many opportunities to be able to contribute to so many projects during my time as an undergraduate student. I work on projects with two primary groups: the Rosenthal Lab based in the Department of Biology at the University of Padua, and the <a href="http://coleoguy.github.io/">Blackmon Lab</a> based in the Department of Biology at Texas A&M University. My work with the Rosenthal Lab focuses on morphometrics and computational forward time simulation of early generation hybrid populations in poecilid fish (<i>Xiphophorus birchmanni</i> and <i>X. malinche</i>), focusing particularly on the interactions between polygenic traits that are targets of mate choice and ecological selection. My work in the Blackmon Lab encompasses a variety of projects studying the evolution of genome structure, with a special emphasis on sex chromosome evolution. I focus primarily on the development and application of phylogenetic comparative methods.<br><br>
</div>

<h3>Drivers of morphometric differentiation in early genetation hybrid <i>Xiphophorus</i> swordtails</h3>
PIs: Drs. <a href="https://www.biologia.unipd.it/en/department/people/teacher-details/?tx_wfqbe_pi1%5Baccount%5D=gil-rosenthal">Gil Rosenthal</a> and <a href="https://www.bio.tamu.edu/faculty-page-heath-blackmon/">Heath Blackmon</a>
<br>
<figure style="float:left;margin-right:20px;margin-bottom:40px;height:300px;width:300px;text-align:center">
        <img src="../images/Xipho hybrids.png" alt="Xiphophorus hybrids">
        <figcaption><i>Xiphophorus hybrids across different environments and time points</i></figcaption>
</figure>
<div>
With advances in sequencing methods corresponding with more access than ever to sequence data, it has become increasingly apparent that naturally occurring hybridization events are significantly more common than they were once thought to be. When two hybridizing species are differentiated along multiple axes, hybrids between them are able to explore a wide range of trait space and populations of these hybrids are diverse in phenotype. When diverse phenotypes are combined with diverse selection regimes, populations differentiate rapidly. The components of selection that drive differentiation in populations following gene flow can often be difficult to tease apart using purely empirical approaches, but computational simulations facilitate easy exploration of a wide variety of selection regimes. If both empirical and computational methods are applied in conjunction, statistical model fitting approaches can then be used to infer the selection regime that best corresponds with phenotypes observed empirically.
</div>
<br>
<figure style="float:right;margin-left:20px;margin-bottom:40px;height:160px;width:300px;text-align:center">
	<img src="../images/Xipho morph.jpg" alt="Xiphophorus morphometric measurements">
        <figcaption><i>Morphometrics collected for stocktank Xiphophorus hybrids</i></figcaption>
</figure>
<div>
My work in the Rosenthal Lab applies these concepts to a system revolving around two species of naturally hybridizing <i>Xiphophorus</i> swordtail fish, <i>X. birchmanni</i> and <i>X. malinche</i>. These sister species are diverged morphologically, physiologically, and behaviorally, with differentiated habitat preferences and mate choice behavior resulting in discordance between the selective pressures encountered by each species. Hybrids between the two species exhibit diverse morphology, physiology, and behavior, representing axes along which hybrid populations can differentiate.  As part of an ongoing project funded under the NSFâ€™s Long Term Research in Environmental Biology grant, the Rosenthal lab monitored replicated mesocosm hybrid populations at regular intervals over a five year period, collecting a variety of morphometric data from individuals in the populations. To complement this extensive empirical dataset, I designed individual-based forward time population simulations that replicate the structure of our empirical populations. These simulations incorporate up to six polygenic traits which have genomic architecture informed by empirical  studies on quantitative trait loci in the system, and implement various combinations of selection functions and strengths, including ecological selection, incompatibility selection, and various forms of mate choice.
</div>













